### PR TITLE
2.x into 3.x

### PR DESCRIPTION
**Pull request by VincentLanglet**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [2.11.2](https://github.com/sonata-project/SonataIntlBundle/compare/2.11.1...2.11.2) - 2021-11-20
+### Fixed
+- [[#488](https://github.com/sonata-project/SonataIntlBundle/pull/488)] Correctly default to the kernel default locale when no locale is provided. ([@VincentLanglet](https://github.com/VincentLanglet))
```
Comment (phansys) [inline CHANGELOG.md:7]:
```suggestion
- [[#488](https://github.com/sonata-project/SonataIntlBundle/pull/488)] Correctly default to the kernel default locale when no locale is provided ([@VincentLanglet](https://github.com/VincentLanglet))
```

**Review by phansys — COMMENTED** [review]
Are the PHPStan / Psalm failures expected?

File: CHANGELOG.md (hunk 1 above)
Comment (jordisala1991) [reply CHANGELOG.md:7]: I guess this should be changed on the branch where it was added, not on a merge.

**Comment by VincentLanglet** [comment]
> Are the PHPStan / Psalm failures expected?

Fixed

File: CHANGELOG.md (hunk 1 above)
Comment (VincentLanglet) [reply CHANGELOG.md:7]: Yeah, if wanted, this should be modified on 2.x

**Review by jordisala1991 — APPROVED** [review]

**Pull request merged**